scripts/release_issue_status/utils.py

```diff
@@ -170,6 +170,10 @@ def run_pipeline(issue_link, sdk_issue_object, pipeline_url):
             "PIPELINE_LINK": {
                 "value": f"{pipeline_url}",
                 "isSecret": False
+            },
+            "SPEC_README":{
+                "value": f"{spec_readme}",
```

Comment:
```suggestion
                "value": spec_readme,
```
I think it should be ok